packages/block-library/src/image/image.js

```diff
@@ -512,7 +512,8 @@ export default function Image( {
 		isSingleSelected &&
 		canEditImage &&
 		! isEditingImage &&
-		! isContentOnlyMode;
+		! isContentOnlyMode &&
+		hasNonContentControls;
```

Comment:
```suggestion
		hasNonContentControls;
```

Do we need to check both `! isContentOnlyMode` and `hasNonContentControls`? Wouldn't `hasNonContentControls` be enough?

I might be misunderstanding something: cc @talldan @jeryj

Reply:
I think you're right @t-hamano 

`blockEditingMode` can only be equal to one value at a time, and there are three possible values: `disabled`, `contentOnly` and `default`. 

But it's a truthy test so, the condition is exclusive: if `hasNonContentControls` is `true`, it automatically implies that `isContentOnlyMode` is `false`.